tsconfig.json

```diff
@@ -2,8 +2,9 @@
   "compilerOptions": {
       "strict": true,
       "esModuleInterop": true,
-      "experimentalDecorators": true
+      "experimentalDecorators": true,
+      "sourceMaps": true,
   },
   "compileOnSave": true,
-  "exclude": ["node_modules", "build"]
+  "exclude": ["node_modules", "build"],
```

Comment:
Please. remove `,` here and on line 6, because trailing commans makes json invalid
`"exclude": ["node_modules", "build"]`